src/portals/admin.js

```diff
@@ -53,13 +53,15 @@ Admin.prototype.goDELETE = function (request, response) {
   var id = this.getId(request.url);
   var self = this;
 
-  if (!id) { return this.notSupported(response); }
-
   function callback (err) {
     if (err) { self.notFound(response); } else { self.noContent(response); }
   }
 
-  this.endpoints.delete(id, callback);
+  if (id) {
+    this.endpoints.delete(id, callback);
+  } else {
```

Comment:
The only change I would ask is to ensure that the request was to `/` to delete everything, instead of deleting everything if an `id` wasn't captured.

Reply:
No worries 👍 

Out of curiosity; what is an example url that might be something without an id? Is it just anything that isn't a number eg. `/test`? Or with multiple directories in the path? eg. `/123/test`. I just want to make sure I cover the right scenarios in a test.

Reply:
Those are good examples. I think the code as-is may actually work correctly (due to multiple checks against `urlPattern`) but for a "delete everything" capability it is good to have tests ensuring it works as intended only on `/` in case other capabilities are added later.

Reply:
Sounds good. I'll sort that out today.

Reply:
Updated